torch_geometric/explain/metrics/faithfulness.py

```diff
@@ -0,0 +1,169 @@
+r"""
+
+
+Faithfulness sore for explainability
+Based on Evaluating Explainability for Graph Neural Networks
+
+
+@misc{https://doi.org/10.48550/arxiv.2208.09339,
+  doi = {10.48550/ARXIV.2208.09339},
+  url = {https://arxiv.org/abs/2208.09339},
+  author = {Agarwal, Chirag and Queen, Owen and Lakkaraju, Himabindu and Zitnik, Marinka},
+  title = {Evaluating Explainability for Graph Neural Networks},
+  code = {https://github.com/mims-harvard/GraphXAI},
+  publisher = {arXiv},
+  year = {2022},
+  copyright = {Creative Commons Attribution 4.0 International}
+
+}
+
+
+we obtain the prediction probability vector ŷ_u using the GNN, i.e. ŷ_u = f (S_u),
+and using the explanation, i.e. ŷ_u′ = f (S_u′), where we generate a masked subgraph S_u′ by only keeping the original
+values of the top-k features identified by an explanation, and get their respective predictions ŷ_u′
+
+Un-Faithfulness of a GNN predictor is given as
+GEF(f,S_u,S_u') = 1 - exp^-KL(ŷ_u || ŷ_u')
+
+Note that higher values indicate a higher degree of unfaithfulness
+"""
```

Comment:
Move the documentation inside the function. Also you can refer to other methods in pyg to see how docs are styled like https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/nn/conv/gcn_conv.py. let me know if you need help with this.